apps/dcos_net/src/dcos_net_mesos_listener.erl

```diff
@@ -142,6 +153,20 @@ terminate(_Reason, _State) ->
 code_change(_OldVsn, State, _Extra) ->
     {ok, State}.
 
+%%%===================================================================
+%%% Internal functions
+%%%===================================================================
+
+reset_state(#state{ref = Ref, timeout_ref = TRef, subs = Subscribers,
+    backoff = Backoff, backoff_timer_ref = BackoffTRef}) ->
```

Comment:
Will do. Though at this point I can't help but wonder if it would be easier just to use `erlang-mode` or some other editor mode as a reference implementation instead of circling back and forth in PRs bringing the code into alignment with the undocumented set of style rules. What do you think? Does it make sense to document the guidelines for all the future references?

Reply:
I totally understand your frustration. The code style standard or document makes sense. However, I'm not sure if it worth the effort (we can discuss the reasons in person). I would love to have a fmt tool, but it doesn't exist. `erlang-mode` is a kinda standard that a very few projects forces and it works only if emacs is the only editor which is not [our] case. Also we have several code styles in one project, some of them are very bad. I'd recommend to use the same code style that is used in the module where you are making changes and also check the style in any big modules that have been added in last couple of years.